### PR TITLE
[doc] Add overview tab documentation in a run single page

### PR DESCRIPTION
**Pull request by arsengit**
- Added overview tab documentation
- Updated single run tabs images in CDN

### CODE REVIEW

**Comment by arsengit** [comment]
Resolves #1506

File: docs/source/ui/pages/run_management.md [hunk 1]
```diff
@@ -67,9 +68,22 @@ Each tab visualizes respective tracked metadata or empty if not tracked.
 - [Figures](#id10)
 - [Settings](#id11)
 
+### Overview Tab
```
Comment (roubkar) [inline docs/source/ui/pages/run_management.md:71]: Maybe just `Overview` without adding `Tab`

File: docs/source/ui/pages/run_management.md [hunk 2]
```diff
@@ -57,6 +57,7 @@ Each training run has a dedicated page on Aim. Use the single run page to observ
 
 Here are the tabs available on the single run page.
 Each tab visualizes respective tracked metadata or empty if not tracked.
+- [Overview](#overview-tab)
```
Comment (roubkar) [inline docs/source/ui/pages/run_management.md:60]: Need to update the link if `Tab` will be removed from the Heading

File: docs/source/ui/pages/run_management.md [hunk 3]
```diff
@@ -67,9 +68,22 @@ Each tab visualizes respective tracked metadata or empty if not tracked.
...
+### Overview Tab
+
+Overview tab shows overall info about the run.
+- These cards can contain information about `Parameters`, `Metrics`, `System Metrics`, `CLI Arguments`, `Environment Variables`, `Packages` and `Git information`. With this data, you can easily reproduce your run.
+
+- Sidebar contains information about  `Run Date`, `Run Duration`, `Run Hash`, attached `Tags` and gives the ability to navigate through tabs.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Single Run page params tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/run-single-page/overview_tab.png">
+
+Also, you can do searching in table, and it will return value by `Match Case`, `Match Word` and `Match Regexp`
```
Comment (roubkar) [inline docs/source/ui/pages/run_management.md:80]: Maybe would be better to say:
Also, you can apply advanced searching/filtering to those card tables.

**Review by roubkar — APPROVED** [review]
lgtm

**Pull request merged**